config/config-example.inc.php

```diff
@@ -2,6 +2,8 @@
 
 mb_internal_encoding('UTF-8');
 
+$routes = json_decode(file_get_contents('routes.json'));
```

Comment:
```suggestion
$routes = json_decode(file_get_contents(__DIR__ . '/routes.json'));
```
This is one more place where the odd behavior of PHP file paths must be accounted for.